cogs/induct.py

```diff
@@ -110,7 +109,7 @@ async def on_member_update(self, before: discord.Member, after: discord.Member)
                 "optional roles like pronouns and year groups\n"
                 "3. Change your nickname to whatever you wish others to refer to you as "
                 "(You can do this by right-clicking your name in the members-list "
-                "to the right & selecting \"Edit Server Profile\").",
+                'to the right & selecting "Edit Server Profile").',
```

Comment:
Prefer strings to use double quotes & escape interior double quotes rather than using single quotes